src/components/Projects.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useEffect } from 'react'
+import styled from 'styled-components/macro'
+import { MainHeaders, OuterWrapper, TitleTertiary, TitleQuaternary, SingleTag, GridDiv, InnerWrapper } from './globalStyling'
+
+export const Projects = () => {
+  const [projectsList, setProjectsList] = useState([])
+
+  const FetchProjects = () => {
+    fetch('https://api.github.com/users/linneaajger/repos')
+      .then((res) => res.json())
+      .then((data) => setProjectsList(data))
+  }
+  // To filter the array of projects and remove the 2 projects with portfolio (index 6-7).
+  // Will need to update this if the index changes with new projects.
+  const FilteredProjects = projectsList.filter((item) => item.name.includes('project-'));
+  FilteredProjects.splice(6, 2);
+
+  // Filtering my starred projects to make them the featured projects
+  const FeaturedProjects = FilteredProjects.filter((project) => project.stargazers_count !== 0)
+  const OtherProjects = FilteredProjects.filter((project) => project.stargazers_count === 0)
```

Comment:
This was such a good solution to this filter-problem! I'm guessing (and hoping) that the star count only applies to the things that you have liked among your own repos.